app/src/main/java/com/joocoding/android/app/githubsearch/service/SearchService.kt

```diff
@@ -0,0 +1,15 @@
+package com.joocoding.android.app.githubsearch.service
+
+import com.joocoding.android.app.githubsearch.model.response.RepositoriesResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface SearchService {
+    @GET("search/repositories")
+    fun searchRepositories(
+        @Query("q") query: String ="mash-up",
```

Comment:
여기 code reformat 해주면 좋을 것 같습니다!  mash-up이 디폴트라니 센스 굳 👍 
```suggestion
        @Query("q") query: String = "mash-up",
```